<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table</class>
 <widget class="QMainWindow" name="Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_All"/>
    <addaction name="actionSave_as_CSV"/>
    <addaction name="actionImport_CSV"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPast"/>
    <addaction name="actionDelet"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSort_A_to_Z"/>
    <addaction name="actionSort_Z_to_A"/>
    <addaction name="actionSort_to_best"/>
    <addaction name="actionSort_to_worse"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToolBar_Save_All"/>
   <addaction name="actionToolBar_Save_as_CVS"/>
   <addaction name="actionToolBar_Export_CVS"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToolBar_Cut"/>
   <addaction name="actionToolBar_Copy"/>
   <addaction name="actionToolBar_Past"/>
   <addaction name="actionToolBar_Delete"/>
  </widget>
  <action name="actionToolBar_Save_All">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/save_all.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionToolBar_Cut">
   <property name="icon">
    <iconset>
     <disabledoff>:/new/prefix1/cut.png</disabledoff>
     <selectedoff>:/new/prefix1/cut_4220.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionToolBar_Copy">
   <property name="icon">
    <iconset>
     <disabledoff>:/new/prefix1/copy.png</disabledoff>
     <selectedoff>:/new/prefix1/copy_paste_documents_1580.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/about.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionToolBar_Past">
   <property name="icon">
    <iconset>
     <disabledoff>:/new/prefix1/paste.png</disabledoff>
     <selectedoff>:/new/prefix1/copy_paste_document_file_1557.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Past</string>
   </property>
  </action>
  <action name="actionToolBar_Delete">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/delete.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/save_all.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSort_A_to_Z">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sort from A to Z</string>
   </property>
  </action>
  <action name="actionSort_Z_to_A">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sort from Z to A</string>
   </property>
  </action>
  <action name="actionSort_to_best">
   <property name="text">
    <string>Sort by ratings (from best)</string>
   </property>
  </action>
  <action name="actionSort_to_worse">
   <property name="text">
    <string>Sort by ratings (from worst)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/exit.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/cut.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionToolBar_Export_CVS">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/exsport_to_csv.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Export CVS</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/copy.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPast">
   <property name="icon">
    <iconset>
     <disabledoff>:/new/prefix1/paste.png</disabledoff>
     <selectedoff>:/new/prefix1/copy_paste_document_file_1557.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Past</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelet">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/delete.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Delet</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionToolBar_Save_as_CVS">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/save_as.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Save as CVS</string>
   </property>
  </action>
  <action name="actionSave_as_CSV">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/save_as.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Save as CSV</string>
   </property>
  </action>
  <action name="actionImport_CSV">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/exsport_to_csv.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Import CSV</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
